### ISSUE TITLE
CTRL+SHIFT+E doesn't work when there is an input in focus

### ISSUE THREAD

**Issue by z0al** [microsoft/vscode#76931]
It appears the default keybinding `ctrl+shift+e` for `View: Show explorer` command doesn't work when there is an input in focus (e.g editor, search bar..etc) instead it writes the `e` letter.

Also if you keep trying it will add `E` (upper case) and the writing behavior becomes broken (e.g. writing random "E"s instead of the letters being typed) but it goes back to normal after pressing backspace enough.

Again, the keybinding works fine if there is no input, for example, switching to the debugger (`ctrl+shift+d`) then immediately try the above keybinding works fine. Also, this **only affects** the snap package, I tried installing the `.deb` directly and it worked fine.

Version: 1.36.0
Commit: 0f3794b38477eea13fb47fbe15a42798e6129338
Date: 2019-07-03T13:20:56.241Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 4.19.0-5-amd64 snap (on Debian 10)

Steps to Reproduce:

1. Open VSCode
2. Focus on an input (e.g. the editor)
3. Press `CTRL+SHIFT+E`

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I can not reproduce this on my Ubuntu.

@z0al can you reproduce if you use vscode insiders
@joaomoreno do you see this? Not sure how it could be connected to the snap package (probably by accident),

**Comment by z0al** [comment]
> can you reproduce if you use vscode insiders

Same problem

> Not sure how it could be connected to the snap package (probably by accident),

Honestly, I don't think it has something to do with the snap package. I have no idea why this is happening but it might have something to do with my machine because I tried it on another Ubuntu 18.04 machine and it works fine.

My setup is Debian 10 (buster) and I'm only experiencing this with VSCode, I tried the same key combination in Chrome and other - electron-based snapped - apps and it works. Going to stick to the traditional Debian package for now.
